source/builtin/src/lib.rs

```diff
@@ -11,6 +11,8 @@
     register_tool(verus),
     register_tool(verifier)
 )]
+#![cfg_attr(verus_keep_ghost, feature(const_trait_impl))]
+#![cfg_attr(verus_keep_ghost, feature(effects))]
```

Comment:
What's this feature?

Reply:
> What's this feature? Rather, why is it needed for this PR?

In the spirit of Rust's regular arithmetic ops, the spec arithmetic ops are defined as trait members (e.g. trait `SpecAdd` has member `spec_add`).  To support declarations like `pub spec const Y: int = 20 - 10;`, implementations of these members need to be declared `const`.  Otherwise, Rust complains that they can't be used in a const declaration like `pub spec const Y: int = 20 - 10;`.  As far as I can tell, Rust does not make it easy to declare implementations of trait functions as `const`.  The best I could find was to use the two features `const_trait_impl` and `effects` (in which you declare traits `#[const_trait]` and impls `impl const`) even though all I need is to declare particular member function implementations const; I don't actually need the generality of `const_trait_impl` and `effects`.  If there's an easier way to do this, let me know.

Reply:
Thanks for the write-up: can you check the progress of those two features and see if it's likely they'll go forward? We don't want to pick up more nightly features if they may be experiments or don't have a path to stabilization.